refarch-integrations/refarch-email-integration/refarch-email-integration-core/src/main/java/de/muenchen/refarch/email/integration/domain/model/BasicMail.java

```diff
@@ -0,0 +1,43 @@
+package de.muenchen.refarch.email.integration.domain.model;
+
+import jakarta.validation.constraints.NotBlank;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class BasicMail {
```

Comment:
can we use records for all models?

Reply:
I looked into that and as far as i understood it's not possible to extends records.
So i would say no.